os/board/Kconfig

```diff
@@ -202,6 +202,15 @@ config ARCH_BOARD
 	default "rtl8721csm"         if ARCH_BOARD_RTL8721CSM
 	default "stm32h745zi-nucleo" if ARCH_BOARD_STM32H745ZI_NUCLEO
 	default "sabre-6quad"	     if ARCH_BOARD_SABRE_6QUAD	
+
+config BOARD_REVISION_VERSION
```

Comment:
We can move this into the RTL8730E Kconfig with RTL8730E prefix.

Reply:
Thanks, I changed